### DISCUSSION TITLE
SimpleLogin has been promising phone number masking since 3 years and it still hasn't been implemented

### DISCUSSION THREAD

**Discussion by AedenThomas** [simple-login/app#2080] [Feature Requests]
@nguyenkims had told that we would be getting [an alpha version at the beginning of 2022](https://github.com/simple-login/app/discussions/555#discussioncomment-1091780)  and we still haven't heard any updates regarding it.

[The X account of SimpleLogin had told phone masking would be coming in 2023](https://x.com/SimpleLogin/status/1629096491791818757?s=20) and still it hasn't come

I am seriously considering switching to Cloaked because it's around the same price as SimpleLogin (just $8 more after discount) and it has all the features of SimpleLogin plus it has phone number masking and a password manager with credit card masking coming soon. 

Hope SimpleLogin can give an update and implement this as soon as possible.

**Comment by nguyenkims** [comment]
Hi for the last 2 years we have been focusing our efforts on SL scalability and reliability as the number of users has significantly increased after we joined Proton. The phone number project isn't discontinued, we've made several attempts at it - an alpha version is actually available for internal users for some time. 

We don't expect to have phone numbers available for all users soon though as it have several challenges:

- expensive
- region based
- most services don't allow VOIP numbers to sign up
- depending on countries, might require personal data like passport, proof of address, etc of the person who uses it

**Reply by mom2louisnchloe-ctrl** [reply]
I have been using a service that is similar to the phone number masking idea via the TextFree Voice app. I pay about $10 per month, and I can "change my phone number" once every 24-hours.  It comes in handy in my line of side-gig work, as I do online mystery shops that requires a different phone number for each shop I perform.

With the TextFree Voice app, I can receive text messages, phone calls, and voicemails, and it even transcribes the voicemails (though not super accurately if the person is speaking fast). 

I am glad to see that I am not the only one who thinks that the phone number masking would be a good idea for SimpleLogin, as I love the free version of the email masking. And being able to add multiple phone numbers in the same way and in the same place that I can email addresses would be amazing - even if they were only available one at a time.

**Reply by katasandi1234** [reply]
> I have been using a service that is similar to the phone number masking idea via the TextFree Voice app. I pay about $10 per month, and I can "change my phone number" once every 24-hours. It comes in handy in my line of side-gig work, as I do online mystery shops that requires a different phone number for each shop I perform.
> 
> With the TextFree Voice app, I can receive text messages, phone calls, and voicemails, and it even transcribes the voicemails (though not super accurately if the person is speaking fast).
> 
> I am glad to see that I am not the only one who thinks that the phone number masking would be a good idea for SimpleLogin, as I love the free version of the email masking. And being able to add multiple phone numbers in the same way and in the same place that I can email addresses would be amazing - even if they were only available one at a time.

what app is this sorry?
can i get the link, i searched " TextFree Voice app" but i see alot of clones.
ALso you said you can change the phone number once very 24hrs, is this included in the 10$ per month subscription or thats seperate?
